src/tokens/RwaTokenFactory.t.sol

```diff
@@ -0,0 +1,88 @@
+/* prettier-disable */
+
+/**
+ * This file is a copy of https://goerli.etherscan.io/address/0xeb7C7DE82c3b05BD4059f11aE8f43dD7f1595bce#code.
+ * The only change is the solidity version, since this repo is using 0.6.x
+ */
+
+////// src/RwaToken.sol
+// Copyright (C) 2020, 2021 Lev Livnev <lev@liv.nev.org.uk>
```

Comment:
We should probably remove Lev's name and list only the Dai Foundation here, since it's a new contract.